### ISSUE TITLE
[Accessibility] Keyboard navigation does not work in the popup

### ISSUE THREAD

**Issue by PinkPandaKatie** [hackademix/noscript#29]
After searching all over, I managed to find the new shortcut for opening the popup: alt-shift-n. However, this does not help at all, since I can't change anything other than the top-level site. I can press left and right to change the settings for top-level, but up and down behave exactly like left and right instead of moving up and down the list of domains like I would expect. I tried every key I can think of, and nothing allows it to select another domain.

It looks like there is no way to use NoScript without a mouse. This is a major accessibility problem for anyone who has difficulty using a mouse or other pointing device.

**Comment by PinkPandaKatie** [comment]
I figured out a workaround - if I press Tab 6 times, then the next row is selected, even though there is no visible indication of this. It would still be much easier if I could press up/down to select different rows instead of blindly pressing tab a bunch of times.

**Comment by adnion** [comment]
Here are the old shortcuts which used to be configurable in about:config (The [FAQs](https://noscript.net/faq) are outdated)
    noscript.keys.toggle: ctrl shift VK_BACK_SLASH.|
    noscript.keys.ui: ctrl shift S
    noscript.keys.tempAllowPage: [no default keyboard shortcut]
    noscript.keys.revokeTemp: [no default keyboard shortcut]

Now only alt-shift-n is [left](https://hackademix.net/2017/11/23/noscript-1012-temporary-allow-all-and-more/)? 
This is an important issue. I would love to see keyboard shortcuts again.

**Comment by woky** [comment]
> I figured out a workaround - if I press Tab 6 times, then the next row is selected

It's 7 times in latest version.

**Comment by maricn** [comment]
On Firefox 72.0b7 (64-bit) on Debian buster it's 8 times now.

Would be great to expose a shortcut for navigation, and/or https://github.com/hackademix/noscript/issues/45.

**Comment by exquo** [comment]
In the current version (NoScript 11) this does work as expected by the original poster: pressing the `up` and `down` keys selects the previous / next domain, the permissions for which can be changed with `left` and `right` arrow keys.

Additionally, the arrow keys can be used to access other parts of NoScript UI: set all to temporarily trusted, disable temporary permissions, etc.

So this issue can probably be closed now. There still remains the issue of adding _better_ keyboard shortcuts (#45), as navigating the menu with arrow keys seems like a workaround.
